Jenkinsfile

```diff
@@ -12,3 +12,4 @@ subsetConfiguration = [ [ jdk: '8',  platform: 'windows', jenkins: null
                       ]
 
 buildPlugin(configurations: subsetConfiguration, failFast: false)
+runBenchmarks('jmh-report.json')
```

Comment:
I had forgotten that Jenkinsfile changes are ignored for anyone that is not a committer to the repository.  It is a safety measure to prevent dangerous changes to the Jenkinsfile.

I'm evaluating the change in my CI infrastructure at home.  Will let you know how it looks in about 45 minutes.